QUEST_LV_0200.tsv

```diff
@@ -7739,87 +7739,87 @@ QUEST_LV_0200_20150714_007742	I should've persuaded him slowly, but I think we w
 QUEST_LV_0200_20150714_007796	The senior monk here is very picky.
-QUEST_LV_0200_20150714_007797	I will gather the monks in front of the entrance of the monastery so tell hunters to gather as well.{nl}After they are gathered, we will unleash the seals that are blocked so that the monsters won't be able to follow.{nl}
+QUEST_LV_0200_20150714_007797	I will gather the monks in front of the entrance of the monastery so tell hunters to gather as well.{nl}After they are gathered, we will unleash seals to block any monsters from following us.{nl}
 QUEST_LV_0200_20150714_007798	But, although we retrieved the monastery, don't expect us to expel Evoniphon.{nl}The father superior will decide everything after he comes out from the Prayer Room.
 QUEST_LV_0200_20150714_007799	Please understand that we could only do this.{nl}We've only listened to your story.
 QUEST_LV_0200_20150714_007800	What Guus said also makes sense.{nl}As they gather in front of the monastery, we should have some decisive evidences that would withdraw Evoniphon's declaration.
 QUEST_LV_0200_20150714_007801	I think we would have enough time.{nl}I am sorry, but please go back to Guus and ask him what happened when Evoniphon was coming.
-QUEST_LV_0200_20150714_007802	You can't prove that he was a cruel assasin.{nl}We can't unlock the ones that are being protected in the castle by just guessing.{nl}
+QUEST_LV_0200_20150714_007802	You can't prove that he was a cruel assasin.{nl}We can't release the ones that are being protected in the castle by just guessing.{nl}
 QUEST_LV_0200_20150714_007803	What they said about the traps and the monsters that conquered the monastery are also the same.{nl}It's only an one-sided story that this was done by Evoniphon.{nl}
 QUEST_LV_0200_20150714_007804	We are the people who pray for the generosity of the Goddesses.{nl}So to withdraw the protection of Evoniphon, you should show them some decisive evidences.
 QUEST_LV_0200_20150714_007805	Hmm. I remember that clearly.{nl}One day, a guy called Evoniphon knocked on the door of the monastery. With his wounded body...{nl}
-QUEST_LV_0200_20150714_007806	After having him in the room, I started to cure him.{nl}He also requested to protect the sanctuary at then.
-QUEST_LV_0200_20150714_007807	That's right. Before he came on that day, one man who seemed to be addicted by the poison also came.{nl}He was in a severe status, so he passed away before I could even ask him a question.{nl}
+QUEST_LV_0200_20150714_007806	After bringing him inside, I began to cure him.{nl}He also requested sanctuary at our monastery.
+QUEST_LV_0200_20150714_007807	That's right. Before he came on that day, one man who seemed to be affected by the poison also came.{nl}He was gravely ill, so he passed away before I could even ask him a question.{nl}
 QUEST_LV_0200_20150714_007808	His grave is in Pasaro Rest Area. With his keepsakes.{nl}Ah, Evoniphon also asked me whether there was someone who came before him.
```

Comment:
Pasaro Rest Area -> Fasadu Rest Area (check if there are more instances for easier replace)